### PR TITLE
Post run feature 

### PR DESCRIPTION
**Pull request by ArmandasRokas**

### CODE REVIEW

File: src/main/java/dk/runs/runners/rest/RunController.java [hunk 1]
```diff
@@ -26,9 +26,10 @@ public Run getRun(@PathVariable int id){
     }
 
     @PostMapping(path = "/createRun")
```
Comment (kpenov16) [inline src/main/java/dk/runs/runners/rest/RunController.java:28]: we do not use verbs in the path, create is defined by a post method so we just use /runs but a new method post

**Review by kpenov16 — CHANGES_REQUESTED** [review]
see RunController

File: src/main/java/dk/runs/runners/rest/RunController.java (hunk 1 above)
Comment (ArmandasRokas) [inline src/main/java/dk/runs/runners/rest/RunController.java:28]: But it works

**Comment by kpenov16** [comment]
need to be fixed

**Pull request merged**